test/unittests/execute_call_test.cpp

```diff
@@ -361,14 +363,14 @@ TEST(execute_call, imported_function_call_void)
 
     const auto module = parse(wasm);
 
-    bool called = false;
-    const auto host_foo = [&called](Instance&, const Value*, int) {
+    static bool called = false;
+    constexpr auto host_foo = [](std::any&, Instance&, const Value*, int) {
```

Comment:
Here it would be better example to pass the `bool called` by `std::any`.

Reply:
I think it's better to have separate tests for explicitly checking context mechanism, and leave the rest of the tests in the simplest form with statics.

So I'll rather add a new test.

Reply:
ok